sematic/db/migrations/20220830211101_persist_resource_reqs.sql

```diff
@@ -0,0 +1,7 @@
+-- migrate:up
+
+ALTER TABLE runs ADD COLUMN resource_requirements_json JSONB;
```

Comment:
Why `_json`?

Reply:
We discussed offline--this is about the need to access python objects via properties, but leave the ORM column storing a string of json.